docker-compose.prod.yml

```diff
@@ -40,3 +48,19 @@ services:
       - api
     depends_on:
       - api
+  postgres:
+    image: postgres:12.0
+    container_name: postgres
+    ports:
+      - "5432:5432"
+    volumes:
+      - /var/data/postgres_data:/var/lib/postgresql/data
```

Comment:
Does this make use of the `postgres_data` volume you create on line 66? Stuff I've seen online seems to use
```yaml
volumes:
  - <volume_name>:<dir>
```

Reply:
I'm pretty sure it does (the /var/data/postgres_data directory exists), but I think I'll updatte it to follow the standard. Thanks!